region/brinstar/kraid.json

```diff
@@ -2313,6 +2223,38 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Leave With Runway",
+                  "notable": false,
+                  "requires": [],
+                  "exitCondition": {
+                    "leaveWithRunway": {
+                      "length": 4,
+                      "openEnd": 1
+                    }
+                  }
+                },
+                {
+                  "name": "Leave Shinecharged",
+                  "notable": false,
+                  "requires": [
+                    {"canShineCharge": {
+                      "usedTiles": 22,
+                      "openEnd": 0
+                    }},
+                    "canShinechargeMovement"
+                  ],
+                  "exitCondition": {
+                    "leaveShinecharged": {
+                      "framesRemaining": 90
+                    }
+                  }
```

Comment:
```suggestion
                  },
                  "devNote": "An X-Mode strat is likely possible but not included. It would require the doors unlocked and canRiskPermanentLossOfAccess."
```